aten/src/ATen/native/cpu/UpSampleKernelAVXAntialias.h

```diff
@@ -578,6 +754,16 @@ void ImagingResampleHorizontalConvolution8u(
       -1, 15, -1, 11, -1, 14, -1, 10, -1, 13, -1, 9, -1, 12, -1, 8
     )
   };
+  const __m256i masks_low_high_c3[2] = {
```

Comment:
It may be worth updating the comment above, which was written with `masks_low_high_c4` in mind?